### DISCUSSION TITLE
error parsing message with type 'tensorflow.GraphDef'

### DISCUSSION THREAD

**Discussion by ccww468** [deepmodeling/deepmd-kit#2738] [General]
I opened a dpmdkit mirror node, and I can't handle it when I use dp test -m, why is that?
![img_v2_847608a6-74b0-4579-bdc0-b7e6e7a0bf3g](https://github.com/deepmodeling/dpgen/assets/140952848/750bc2c2-c3bb-4913-815d-d13b7fba319f)
![img_v2_c6a0be64-ee74-4f73-9f61-a4fbdd09ca9g](https://github.com/deepmodeling/dpgen/assets/140952848/bfa92bce-fe28-4e25-82e7-f45fd6dff234)

**Comment by njzjz** [comment]
Several reasons can cause this error message. See #790 and #1550.

It would be helpful if you could provide this file.
